web/lib/api/metrics/totalRequests.ts

```diff
@@ -26,7 +26,6 @@ export async function getTotalRequests(
     }
   );
   const query = `
-
```

Comment:
Let's try to commit these in the future, because then it will be harder to look in the history for this specific file for issues

Reply:
sry it's from my previous pr

Reply:
no worries